src/render/painter.js

```diff
@@ -188,15 +188,15 @@ class Painter {
         this._backgroundTiles = {};
     }
 
-    updateTerrain(style: Style, cameraChanging: boolean) {
+    updateTerrain(style: Style, isDragging: boolean) {
```

Comment:
Nit: Consider changing the name of `isDragging` to be more descriptive of the behavior? e.g. `keepCameraDistance`?